apps/designer/.env.example

```diff
@@ -26,3 +26,12 @@ ASSET_CDN_URL=
 PUBLISHER_ENDPOINT=
 PUBLISHER_HOST=
 PUBLISHER_TOKEN=
+
+# Base URL for accessing user-generated content (canvas / preview / etc)
+# Should contatin protocol and host (e.g. https://example.com)
+# Will be detected automatically in development environments like localhost or preview deployment
+USER_CONTENT_BASE_URL=
```

Comment:
How about BUILD_ORIGIN

<img width="275" alt="Screenshot 2022-10-11 at 18 08 04" src="https://user-images.githubusercontent.com/52824/195143669-5d984a0e-2a08-4992-9a51-d43fc0f1301e.png">

Reply:
No strong opinion, but USER_CONTENT describes the intent better in my view. 

As for ORIGIN instead of BASE_URL, need to check definition of ORIGIN, but probably a good idea.

Reply:
the problem with user content is that this is the second term that describes user's build, we need to agree on one, unless you see a difference between user content and build